bin/check-consul-kv-ttl.rb

```diff
@@ -95,17 +94,28 @@ class CheckConsulKvTTL < Sensu::Plugin::Check::CLI
          proc: proc { |a| a.to_i },
          default: 60
 
+  option :token,
+         description: 'ACL token',
+         long: '--token ACL_TOKEN'
+
   # Do work
   def run
     Diplomat.configure do |dip|
       dip.url = config[:consul]
+      dip.acl_token = config[:token]
     end
 
     begin
       # Retrieve the kv
       data = Diplomat::Kv.get(config[:kv])
-    rescue Faraday::ResourceNotFound
-      critical "Key/Value(#{config[:kv]}) pair does not exist in Consul."
+    rescue Faraday::ResourceNotFound, Diplomat::KeyNotFound
+      critical %(Key/value pair "#{config[:kv]}" does not exist in Consul.)
+    rescue Diplomat::UnknownStatus => e
+      if e.message.include?('403')
+        critical %(ACL token is not authorized to access "#{config[:kv]}")
```

Comment:
is there any reason to use `%(my string with a "#{var}")` over `ciritical "my string with a #{var}"`? The reason I ask is that this is not the most common way people define strings in ruby and in general in the sensu plugins they prefer standard ruby string interpolation. I also see that you are using the standard in other locations. I think we should either use the same pattern everywhere or if there is some nuance I missed it needs a comment above it.

My guess is that you want the literal double quotes in there, if thats the case please use `%Q[string]` or `%Q(string)` over `%(string)`. I realize this is the default (same as not specifying) but I prefer to be more than less explicit. I see `%Q()` pretty frequently but I had to look up what the default was when not specified as I don't see it without a modifier frequently.

Reply:
To be transparent, I went with Rubocop rules specified in this repo once I was done so `%()` came from the rules.

Indeed here I'd like to have the double quotes and I'm okay with whatever syntax you want.

Just let me know exactly which one you prefer because %Q() won't pass the rules in Rubocop.

Reply:
interesting, I will look at this tonight and see if I can figure out why thats not legit according to rubocop.